### DISCUSSION TITLE
Backup of the Blockchain

### DISCUSSION THREAD

**Discussion by Celt1967** [Chia-Network/chia-blockchain#6787] [General]
Is it possible to just simply copy the ../mainnet/db folder to make a backup of the blockchain so that it can be restored if there are any issues; thus saving having to start from 0 ?

**Comment by loppefaaret** [comment]
yes, but make sure the client is closed down when you do take this backup!

**Reply by Celt1967** [reply]
Could you explain why please? Seemed to work fine the other day.

**Reply by loppefaaret** [reply]
if the full_node process writes new data to the database, while you are copying it out, it very easily gets corrupted

**Reply by Celt1967** [reply]
Ahh ok thank you.
